src/Amazon.DynamoDb.Tests/Actions/DbValueTests.cs

```diff
@@ -298,11 +304,44 @@ public void DbMap3()
 		public void DbList2()
 		{
 
-			var value = DbValue.FromJson(JsonObject.Parse(@"{ ""L"": [ { ""N"": ""1.1"" }, { ""N"":""7.543"" } ] }"));
+			var value = System.Text.Json.JsonSerializer.Deserialize<DbValue>(@"{ ""L"": [ { ""N"": ""1.1"" }, { ""N"":""7.543"" } ] }");
 
 			Assert.Equal(DbValueType.L, value.Kind);
 
 			Assert.Equal(new[] { 1.1f, 7.543f }, value.ToArray<float>());
 		}
-	}
+
+        
+    }
+
+    public static class DynamoTestHelper

```

Comment:
This is the thing that belongs in its own file, but wouldn't work when in its own file...